### PR TITLE
LD_PRELOAD should use colon rather then space as delimiter

### PR DESCRIPTION
**Pull request by miffedmap**
#508

### CODE REVIEW

**Comment by lethosor** [comment]
According to #276 and several online manpages ([this](http://manpages.ubuntu.com/manpages/utopic/man8/ld.so.8.html) and [this](http://lmbench.sourceforge.net/cgi-bin/man?keyword=ld.so&section=8), to name a couple), spaces shouldn't be a problem.

File: package/linux/dfhack [hunk 1]
```diff
@@ -38,7 +38,7 @@ old_tty_settings=$(stty -g)
 
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:"./hack/libs":"./hack"
 
-PRELOAD_LIB="$PRELOAD_LIB ./hack/libdfhack.so"
+PRELOAD_LIB="$PRELOAD_LIB:/hack/libdfhack.so"
```
Comment (lethosor) [inline package/linux/dfhack]: Should still be `./hack/libdfhack.so`.

Comment (miffedmap) [inline package/linux/dfhack]: all thumbs here.

**Comment by miffedmap** [comment]
Yeah, but I haven't tested it with multiple entries having dirnames containing spaces and I'm guessing that wouldn't work properly.

Pushed the missing period -- you have sharp eyes.

**Pull request merged**